### ISSUE TITLE
GUI: Head not turning anymore

### ISSUE THREAD

**Issue by chris-hld** [SoundScapeRenderer/ssr#135]
The latest master commit is not updating the GUI head orientation, when connected to a head tracker.
Updating the TCP interface and sound output works, it's only the GUI.

**Comment by mgeier** [comment]
I guess the reason is that I changed the trackers to manipulate the "reference offset" instead of the "reference". See "collateral changes" in #127.

Ideally the head in the GUI should show both pieces of information, but I'm not sure how this can be achieved.

For loudspeaker-based renderers, there is a little cross symbol that shows the reference offset.

**Comment by chris-hld** [comment]
Ah, yes, that might be it!
I agree with the change. Even if listeners turn their head, the scene coordinate system stays the same.
But in this case the GUI Head should reflect the "reference offset"!

**Comment by chris-hld** [comment]
BTW, there is also a little cross on top of the GUI head. So maybe it is even possible to show both?
![image](https://user-images.githubusercontent.com/10830393/52013059-69ec2700-24dc-11e9-8c4e-c9f3fc8d4822.png)

**Comment by JensAhrens** [comment]
This cross marks the origin of the global coordinate system. It is not movable. 

In qopenglplotter.cpp, there is an [if/else statement](https://github.com/SoundScapeRenderer/ssr/blob/master/src/gui/qopenglplotter.cpp#L373):

```
if (_plot_listener)
{ 
    (...)
}
else
{
    (...)
    // draw cross (showing the reference offset)
    (...)
}
```

This would needed to be changed (it's [here](https://github.com/SoundScapeRenderer/ssr/blob/master/src/gui/qopenglplotter.cpp#L453)) for the reference offset also to be visible when a listener is apparent.

**Comment by mgeier** [comment]
I think this has been fixed in #358 and #360.

**Comment by HaHeho** [comment]
> I think this has been fixed in #358 and #360.

The latest release indeed fixed such an issue with a Polhemus tracker for me.

**Comment by chris-hld** [comment]
Closing since seems to be fixed in the meantime.
